pkgs/development/php-packages/datadog_trace/default.nix

```diff
@@ -0,0 +1,20 @@
+{ buildPecl, curl, fetchurl, lib, pcre2 }:
+
+buildPecl rec {
+  pname = "ddtrace";
+  version = "0.70.0";
+
+  src = fetchurl {
+    url = "http://pecl.php.net/get/datadog_trace-${version}.tgz";
+    sha256 = "s1uJNAP7LI6FmNIcitCJaoRxTgceWOopIHQH9PkZ6vI=";
+  };
```

Comment:
`ddtrace.so` is what's built, hence the pname. Pecl has the project as `datadog_trace-...` so we include a `src = fetchurl` to overwrite satisfy the pattern in the url